docs/user_guide/transformation/PowerTransformer.rst

```diff
@@ -5,66 +5,349 @@
+
+Power transformations differ from scalers in that they modify the distribution of the data, typically to stabilize variance and normalize the distribution, whereas scalers simply adjust the scale of the data without altering its underlying distribution.
+
+In short, power functions provide an excellent data analysis toolkit, especially for linear models (regression or classification).
+
+Special cases of power transformations
+--------------------------------------
+
+Most variable transformations, like the logarithm, the reciprocal and the square root, are special cases of power transformations, where the exponent (lambda) is 0, -1 and 0.5, respectively.
+
+You can apply these transformations with :class:`PowerTransformer()`, as we will see later in this page, or through dedicated transformers, like :class:`LogarithmTransformer` and :class:`ReciprocalTransformer`.
+
+
+Which lambda should I choose?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The challenge of power transformations resides in finding the right lambda for the transformation. In general, this consists of trial and error, or using generalization functions like the Box-Cox or the Yeo-Johnson transformation.
+
+As general guidelines, if the variables are right-skewed we'd use lambda <1, and if the variables are left-skewed we'd use lambda >1.
+
+Box-Cox transformation
+~~~~~~~~~~~~~~~~~~~~~~
+
+The Box-Cox transformation is a generalization of power transformations that finds an optimal lambda to stabilize variance and make the data more normally distributed. This transformation only accepts positive values.
+
+Feature-engine's :class:`BoxCoxTransformer()` applies the Box-Cox transformation.
+
+Yeo-Johnson transformation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Yeo-Johnson transformation extends the Box-Cox transformation to handle both positive and negative values, to find an optimal lambda for the transformation.
+
+:class:`YeoJohnsonTransformer()` applies the Yeo-Johnson transformation.
+
+
+Other transformers
+~~~~~~~~~~~~~~~~~~
+
+Feature-engine also provides the following power transformers:
+
+- :class:`LogTransformer`
+- :class:`LogCpTransformer`
+- :class:`ReciprocalTransformer`
+- :class:`ArcsinTransformer`
+
+For more details about these variance stabilizing transformations, check the article `Variance stabilizing transformations in machine learning <https://www.blog.trainindata.com/variance-stabilizing-transformations-in-machine-learning/>`.
+
+
+Python example
+--------------
+
+:class:`PowerTransformer()` applies power transformations to numerical independent variables. We'll use the Ames House Prices' dataset to see it in action. First, let's load the dataset and split it into train and test sets:
 
 .. code:: python
```

Comment:
one : not 2

Reply:
I didn't get this. Where should it be one, not 2?

Reply:
Oh, ok! The error was due to ..code instead of .. code (with space between .. and code). Thx!